### PR TITLE
add e2e test for tracking request

### LINKED ISSUE #600: assert xhr request in E2E test
# TL;DR

need checking tracker request in e2e test

### PR DESCRIPTION
**Pull request by shoota**
# TL;DR

Closes #600

### How to check

*   checkout this pr.
*   yarn install && yarn build
*   run `npm start`
*   run `yarn e2e` in another console

### Check list

*   [ ] passed tests

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,60 +0,0 @@
-FROM ubuntu:bionic-20180426
```
Comment (shoota) [inline Dockerfile]: use circleci image

File: examples/kaizen-platform/index.test.ts [hunk 2]
```diff
@@ -1,24 +0,0 @@
-import { NightwatchAPI } from 'nightwatch'
```
Comment (shoota) [inline examples/kaizen-platform/index.test.ts]: all *.test.ts move under test/

File: examples/nightwatch.json [hunk 3]
```diff
@@ -1,48 +0,0 @@
-{
```
Comment (shoota) [inline examples/nightwatch.json]: good bye nightwatch

File: examples/test/tracker/built-in.test.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+import { RequestLogger } from 'testcafe'
+import { baseUrl, envRegex } from '../../testcafe-conf'
+import { query } from '../test-util'
+
+const envLogger = RequestLogger(envRegex)
+const l = `${baseUrl}/built-in/`
+
+fixture('basic tracker').page(l)
+
+test.requestHooks(envLogger)('environment request', async (t) => {
```
Comment (shoota) [inline examples/test/tracker/built-in.test.ts]: new e2e test for tracking request

File: examples/test/tracker/kainzen-platform-gtm.test.ts [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+const l = `${deployBaseUrl}/kaizen-platform/`
+
+fixture('basic tracker').page(l)
+
+test.requestHooks(intLogger)('tracking request', async (t) => {
+  await t.wait(5000)
+  assert.ok(intLogger.requests.length > 0)
+  const intRecord = intLogger.requests[0]
+  assert.equal(intRecord.request.method, 'get')
+  const event = query(intRecord)['e'].split(',')
```
Comment (shoota) [inline examples/test/tracker/kainzen-platform-gtm.test.ts]: event object can do test!!

File: examples/test/tracker/built-in.test.ts (hunk 4 above)
Comment (dkimura) [reply examples/test/tracker/built-in.test.ts]: 🎉

**Pull request merged**